### PR TITLE
async version of export html hook

### PR DESCRIPTION
**Pull request by JohnMcLear**

### CODE REVIEW

**Comment by JohnMcLear** [comment]
Try this @Gared

File: src/node/utils/ExportHtml.js [hunk 1]
```diff
@@ -420,14 +420,11 @@ function getHTMLFromAtext(pad, atext, authorColors)
...
-      hooks.aCallAll("asyncLineHTMLForExport", context, function(err, newLineFunction){
-        // For each function returned by the hook call
-        // Process the text based on the function
-        newLineFunction.forEach(function(fn){
-          context.lineContent = fn(context); // note the fn
-        });
-        // We now have a line that has been processed by each hook function
-        lineContent = context.lineContent; // modified lineContent here
+      hooks.aCallAll("asyncLineHTMLForExport", context, function(err, lineContents){
+        if(lineContents[1]){
```
Comment (Gared) [inline src/node/utils/ExportHtml.js]: Why do you take the second element of this array?

Comment (JohnMcLear) [inline src/node/utils/ExportHtml.js]: Yea this is an error, fixing up now

Comment (JohnMcLear) [inline src/node/utils/ExportHtml.js]: I'm being dumb here and can't figure out how to fix this..  Sunday issues..

**Pull request closed without merging**